delivery_api_client/api/__init__.py

```diff
@@ -3,4 +3,4 @@
 # flake8: noqa
 
 # import apis into api package
-from openapi_client.api.delivery_api import DeliveryApi
+from delivery_api_client.api.delivery_api import DeliveryApi
```

Comment:
I'm curious to know why it was called `openapi_client` earlier instead of `delivery_api_client`.

Reply:
That was just the default name given by the codegen tool (see codegeneration/package.json), but changed the name to be more in line with what our other SDKs use.